tools/src/main/java/org/apache/pdfbox/tools/AddText.java

```diff
@@ -47,13 +47,15 @@ public String writeText(String annotation) throws IOException
         if (!doc.isEncrypted()) {
             
             PDFTextStripper stripper = new PDFTextStripper();
+            stripper.setStartPage(a_page+1);
```

Comment:
I assume the start and end page are the same page because you only want to edit one page, but I'm not entirely clear on the functionality of text strippers so I would appreciate some clarification

Reply:
Text strippers read the text from a range of pages. Since we can only annotate one page at a time, we want to read a range of only 1 page, which done by setting the start page and the end page to the same page.

Reply:
I thought that was it, thank you for the clarification